glyphs-reader/src/font.rs

```diff
@@ -557,15 +604,18 @@ fn custom_param_mut<'a>(
     None
 }
 
-fn glyphs_v2_field_name_and_default(nth_axis: usize) -> Result<(&'static str, f64), Error> {
+fn glyphs_v2_field_name_and_default(
+    nth_axis: usize,
+) -> Result<(&'static [&'static str], f64), Error> {
     // Per https://github.com/googlefonts/fontmake-rs/pull/42#pullrequestreview-1211619812
     // the field to use is based on the order in axes NOT the tag.
     // That is, whatever the first axis is, it's value is in the weightValue field. Long sigh.
     // Defaults per https://github.com/googlefonts/fontmake-rs/pull/42#discussion_r1044415236.
+    // v2 instances use novel field names so send back several for linear probing.
     Ok(match nth_axis {
-        0 => ("weightValue", 100_f64),
-        1 => ("widthValue", 100_f64),
-        2 => ("customValue", 0_f64),
+        0 => (&["weightValue", "interpolationWeight"], 100_f64),
```

Comment:
the compiler should be able to infer the float type, here?

Reply:
yup, ty